server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -5199,6 +5209,10 @@ public AsyncIOProcessor<Translog.Location> getTranslogSyncProcessor() {
         return translogSyncProcessor;
     }
 
+    public boolean ongoingEngineMigration() {
+        return RemoteStoreUtils.isMigrationDirectionSet(clusterService) == true;
```

Comment:
This will return true for all the shards during migration . Will that have any side effects ?

Reply:
Removed this altogether from `IndexShard`